service/svcutil/svcutil_test.go

```diff
@@ -170,6 +171,55 @@ func Test_ExecuteTask_Logger(t *testing.T) {
 	}
 }
 
+func Test_ExecuteTask_ConsoleLogger(t *testing.T) {
+	// Arrange
+	tmpDir, testExe := makeHelloForeverExe(t)
+	defer os.RemoveAll(tmpDir)
+
+	var logBuf bytes.Buffer
+	var errorlogBuf bytes.Buffer
+
+	taskConf := svcutil.TaskConfig{
+		Path:             testExe,
+		ExecTimeout:      2 * time.Second,
+		GracefulShutDown: 1 * time.Second,
+		Logger:           log.New(&logBuf, "", 0),
+		ErrorLogger:      log.New(&errorlogBuf, "", 0),
+	}
+
+	// Act
+	svcutil.ExecuteTask(nil, taskConf)
+
+	// Assert
+	output := logBuf.String()
+	if len(output) == 0 {
+		t.Fatalf("Expected some logging output")
+	}
+	fmt.Println("output:")
+	fmt.Println(output)
```

Comment:
The above two lines would not be important if the test was successful, but it would be helpful if the test failed.
So, how about outputting the contents of the logging buffer as part of the test failure message in the following two `t.Errorf`?
It would be the same for `erroroutput` as well.